tests/cuda_test/test_print_deallocate.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=protected-access
```

Comment:
What is this pylint error? Why are you disabling it?

Reply:
The pylint error `# pylint: disable=protected-access` is related to accessing a protected member. I am disabling it to test the `_print_Deallocate` method of the printer class which is a protected method.

Reply:
This shouldn't be necessary. You should be able to do `printer.doprint(expr)` for the same result

Reply:
Updated here 7f36c249828e9d131eb20db0e35963811350519c

Reply:
Thanks :) but is this file still needed?
If the code is tested properly in the other tests then I think that is enough. Those tests are more complete anyway as they also check that the code compiles